root/static/scripts/edit/MB/TextList.js

```diff
@@ -18,13 +18,16 @@ MB.Form.TextList = function (input) {
     var $template = $('.' + template.replace(/\./g, '\\.'));
     var counter = 0;
 
-    self.removeEvent = function () {
+
+    self.removeEvent = function (event) {
```

Comment:
This adds an extra line for some reason, plus (event) which seems unused (I suspect it got removed in a separate PR and you're just readding it by accident).